consoleProject fails to compile compiler bridge for scala 2.13

## steps

sbt version: 1.3.3

```
sbt:sbt-console-project-bug> set scalaVersion := "2.13.1"
[info] Defining scalaVersion
[info] The new value will be used by Compile / compileBinaryFileInputs, Compile / compileSourceFileInputs and 28 others.
[info] 	Run `last` for details.
[info] Reapplying settings...
[info] Set current project to sbt-console-project-bug (in build file:/Users/jason/source/temp/sbt-console-project-bug/)
sbt:sbt-console-project-bug> consoleProject
[info] Non-compiled module 'compiler-bridge_2.13' for Scala 2.12.10. Compiling...
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/Compat.scala:17: error: object shell is not a member of package scala.tools.nsc.interpreter
import scala.tools.nsc.interpreter.shell.ReplReporterImpl
                                   ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/Compat.scala:26: error: not found: type ReplReporterImpl
    new ReplReporterImpl(settings, writer)
        ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:16: error: object shell is not a member of package scala.tools.nsc.interpreter
import scala.tools.nsc.interpreter.shell.{ ILoop, ShellConfig, ReplReporterImpl }
                                   ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:45: error: not found: type ILoop
    val loop = new ILoop(ShellConfig(interpreterSettings)) {
                   ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:48: error: not found: type ReplReporterImpl
          val reporter = new ReplReporterImpl(interpreterSettings)
                             ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:49: error: not found: value intp
          intp = new IMain(interpreterSettings, reporter) {
          ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:54: error: not found: value intp
          intp.setContextClassLoader()
          ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:56: error: value createInterpreter is not a member of AnyRef
          super.createInterpreter(interpreterSettings)
                ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:59: error: not found: value intp
          intp.beQuietDuring {
          ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:60: error: not found: value intp
            intp.bind(id, value.asInstanceOf[AnyRef].getClass.getName, value)
            ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:66: error: not found: value intp
          intp.interpret(initialCommands)
          ^
/var/folders/_l/q8t5_1gj5yx49v31fk6mr1p40000gn/T/sbt_df685972/xsbt/ConsoleInterface.scala:73: error: not found: value intp
          intp.interpret(cleanupCommands)
          ^
12 errors found
[info] Attempting to fetch org.scala-sbt:compiler-bridge_2.13:1.3.1.
[error] (run-main-0) Error compiling the sbt component 'compiler-bridge_2.13'
[error] Error compiling the sbt component 'compiler-bridge_2.13'
[error] 	at sbt.internal.inc.AnalyzingCompiler$.handleCompilationError$1(AnalyzingCompiler.scala:342)
[error] 	at sbt.internal.inc.AnalyzingCompiler$.$anonfun$compileSources$4(AnalyzingCompiler.scala:359)
[error] 	at sbt.internal.inc.AnalyzingCompiler$.$anonfun$compileSources$4$adapted(AnalyzingCompiler.scala:354)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:489)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:499)
[error] 	at sbt.internal.inc.AnalyzingCompiler$.$anonfun$compileSources$2(AnalyzingCompiler.scala:354)
[error] 	at sbt.internal.inc.AnalyzingCompiler$.$anonfun$compileSources$2$adapted(AnalyzingCompiler.scala:346)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:489)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:499)
[error] 	at sbt.internal.inc.AnalyzingCompiler$.compileSources(AnalyzingCompiler.scala:346)
[error] 	at sbt.internal.inc.ZincComponentCompiler.$anonfun$compileAndInstall$3(ZincComponentCompiler.scala:261)
[error] 	at sbt.internal.inc.ZincComponentCompiler.$anonfun$compileAndInstall$3$adapted(ZincComponentCompiler.scala:250)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:489)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:499)
[error] 	at sbt.internal.inc.ZincComponentCompiler.$anonfun$compileAndInstall$2(ZincComponentCompiler.scala:250)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at sbt.internal.util.BufferedLogger.bufferQuietly(BufferedLogger.scala:119)
[error] 	at sbt.internal.inc.ZincComponentCompiler.$anonfun$compileAndInstall$1(ZincComponentCompiler.scala:250)
[error] 	at sbt.internal.inc.ZincComponentCompiler.$anonfun$compileAndInstall$1$adapted(ZincComponentCompiler.scala:247)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:489)
[error] 	at sbt.io.IO$.withTemporaryDirectory(IO.scala:499)
[error] 	at sbt.internal.inc.ZincComponentCompiler.compileAndInstall(ZincComponentCompiler.scala:247)
[error] 	at sbt.internal.inc.ZincComponentCompiler.$anonfun$compiledBridgeJar$1(ZincComponentCompiler.scala:215)
[error] 	at sbt.internal.inc.IfMissing$Define.run(IfMissing.scala:19)
[error] 	at sbt.internal.inc.ZincComponentManager.createAndCache$1(ZincComponentManager.scala:49)
[error] 	at sbt.internal.inc.ZincComponentManager.$anonfun$files$3(ZincComponentManager.scala:60)
[error] 	at sbt.internal.inc.ZincComponentManager.getOrElse$1(ZincComponentManager.scala:42)
[error] 	at sbt.internal.inc.ZincComponentManager.$anonfun$files$2(ZincComponentManager.scala:60)
[error] 	at sbt.internal.inc.ZincComponentManager$$anon$1.call(ZincComponentManager.scala:89)
[error] 	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:95)
[error] 	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:80)
[error] 	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:99)
[error] 	at xsbt.boot.Using$.withResource(Using.scala:10)
[error] 	at xsbt.boot.Using$.apply(Using.scala:9)
[error] 	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:60)
[error] 	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:50)
[error] 	at xsbt.boot.Locks$.apply0(Locks.scala:31)
[error] 	at xsbt.boot.Locks$.apply(Locks.scala:28)
[error] 	at sbt.internal.inc.ZincComponentManager.lock(ZincComponentManager.scala:89)
[error] 	at sbt.internal.inc.ZincComponentManager.$anonfun$lockSecondaryCache$1(ZincComponentManager.scala:86)
[error] 	at scala.Option.map(Option.scala:230)
[error] 	at sbt.internal.inc.ZincComponentManager.lockSecondaryCache(ZincComponentManager.scala:86)
[error] 	at sbt.internal.inc.ZincComponentManager.fromSecondary$1(ZincComponentManager.scala:58)
[error] 	at sbt.internal.inc.ZincComponentManager.$anonfun$files$6(ZincComponentManager.scala:64)
[error] 	at sbt.internal.inc.ZincComponentManager.getOrElse$1(ZincComponentManager.scala:42)
[error] 	at sbt.internal.inc.ZincComponentManager.$anonfun$files$5(ZincComponentManager.scala:64)
[error] 	at sbt.internal.inc.ZincComponentManager$$anon$1.call(ZincComponentManager.scala:89)
[error] 	at xsbt.boot.Locks$GlobalLock.withChannel$1(Locks.scala:95)
[error] 	at xsbt.boot.Locks$GlobalLock.xsbt$boot$Locks$GlobalLock$$withChannelRetries$1(Locks.scala:80)
[error] 	at xsbt.boot.Locks$GlobalLock$$anonfun$withFileLock$1.apply(Locks.scala:99)
[error] 	at xsbt.boot.Using$.withResource(Using.scala:10)
[error] 	at xsbt.boot.Using$.apply(Using.scala:9)
[error] 	at xsbt.boot.Locks$GlobalLock.ignoringDeadlockAvoided(Locks.scala:60)
[error] 	at xsbt.boot.Locks$GlobalLock.withLock(Locks.scala:50)
[error] 	at xsbt.boot.Locks$.apply0(Locks.scala:31)
[error] 	at xsbt.boot.Locks$.apply(Locks.scala:28)
[error] 	at sbt.internal.inc.ZincComponentManager.lock(ZincComponentManager.scala:89)
[error] 	at sbt.internal.inc.ZincComponentManager.lockLocalCache(ZincComponentManager.scala:82)
[error] 	at sbt.internal.inc.ZincComponentManager.files(ZincComponentManager.scala:64)
[error] 	at sbt.internal.inc.ZincComponentManager.file(ZincComponentManager.scala:70)
[error] 	at sbt.internal.inc.ZincComponentCompiler.compiledBridgeJar(ZincComponentCompiler.scala:215)
[error] 	at sbt.internal.inc.ZincComponentCompiler$ZincCompilerBridgeProvider.compiledBridge(ZincComponentCompiler.scala:75)
[error] 	at sbt.internal.inc.ZincComponentCompiler$ZincCompilerBridgeProvider.fetchCompiledBridge(ZincComponentCompiler.scala:81)
[error] 	at sbt.internal.inc.AnalyzingCompiler.loader(AnalyzingCompiler.scala:260)
[error] 	at sbt.internal.inc.AnalyzingCompiler.getInterfaceClass(AnalyzingCompiler.scala:278)
[error] 	at sbt.internal.inc.AnalyzingCompiler.call(AnalyzingCompiler.scala:245)
[error] 	at sbt.internal.inc.AnalyzingCompiler.consoleCommandArguments(AnalyzingCompiler.scala:234)
[error] 	at sbt.internal.inc.AnalyzingCompiler.console(AnalyzingCompiler.scala:188)
[error] 	at sbt.Console.console0$1(Console.scala:48)
[error] 	at sbt.Console.$anonfun$apply$2(Console.scala:51)
[error] 	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
[error] 	at sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
[error] 	at sbt.TrapExit$App.run(TrapExit.scala:257)
[error] 	at java.lang.Thread.run(Thread.java:748)
[error] Nonzero exit code: 1
[error] (consoleProject) Nonzero exit code: 1
[error] Total time: 3 s, completed 28/11/2019 7:20:04 PM
```

## problem

`consoleProject` fails with the above error when using Scala 2.13.

## expectation

`consoleProject` should start the REPL.

## notes

This line in the output seems odd:
```
[info] Non-compiled module 'compiler-bridge_2.13' for Scala 2.12.10. Compiling...
```

--- a/main/src/main/scala/sbt/Defaults.scala
+++ b/main/src/main/scala/sbt/Defaults.scala
@@ -488,6 +488,9 @@ object Defaults extends BuildCommon {
     clean := clean.dependsOn(cleanIvy).value,
     scalaCompilerBridgeBinaryJar := None,
     scalaCompilerBridgeSource := ZincLmUtil.getDefaultBridgeModule(scalaVersion.value),
+    consoleProject / scalaCompilerBridgeSource := ZincLmUtil.getDefaultBridgeModule(
+      appConfiguration.value.provider.scalaProvider.version
+    ),
   )
   // must be a val: duplication detected by object identity
   private[this] lazy val compileBaseGlobal: Seq[Setting[_]] = globalDefaults(

--- a/main/src/main/scala/sbt/internal/ConsoleProject.scala
+++ b/main/src/main/scala/sbt/internal/ConsoleProject.scala
@@ -25,7 +25,7 @@ object ConsoleProject {
     val (state1, dependencyResolution) =
       extracted.runTask(Keys.dependencyResolution, state)
     val (_, scalaCompilerBridgeBinaryJar) =
-      extracted.runTask(Keys.scalaCompilerBridgeBinaryJar, state1)
+      extracted.runTask(Keys.scalaCompilerBridgeBinaryJar.in(Keys.consoleProject), state1)
     val scalaInstance = {
       val scalaProvider = state.configuration.provider.scalaProvider
       ScalaInstance(scalaProvider.version, scalaProvider.launcher)
@@ -50,7 +50,8 @@ object ConsoleProject {
           componentProvider = app.provider.components,
           secondaryCacheDir = Option(zincDir),
           dependencyResolution = dependencyResolution,
-          compilerBridgeSource = extracted.get(Keys.scalaCompilerBridgeSource),
+          compilerBridgeSource =
+            extracted.get(Keys.scalaCompilerBridgeSource.in(Keys.consoleProject)),
           scalaJarsTarget = zincDir,
           classLoaderCache = state1.get(BasicKeys.classLoaderCache),
           log = log

--- a/sbt/src/sbt-test/console/project-compiler-bridge/build.sbt
+++ b/sbt/src/sbt-test/console/project-compiler-bridge/build.sbt
@@ -2,4 +2,4 @@ scalaVersion := "2.13.1"
 
 // Send some bogus initial command so that it doesn't get stuck.
 // The task itself will still succeed.
-Compile / consoleProject / initialCommands := "bail!"
+consoleProject / initialCommands := "bail!"

--- a/sbt/src/sbt-test/console/project-compiler-bridge/build.sbt
+++ b/sbt/src/sbt-test/console/project-compiler-bridge/build.sbt
@@ -1,0 +1,5 @@
+scalaVersion := "2.13.1"
+
+// Send some bogus initial command so that it doesn't get stuck.
+// The task itself will still succeed.
+Compile / consoleProject / initialCommands := "bail!"

--- a/sbt/src/sbt-test/console/project-compiler-bridge/project/build.sbt
+++ b/sbt/src/sbt-test/console/project-compiler-bridge/project/build.sbt
@@ -1,0 +1,1 @@
+scalaVersion := "2.12.10"

--- a/sbt/src/sbt-test/console/project-compiler-bridge/test
+++ b/sbt/src/sbt-test/console/project-compiler-bridge/test
@@ -1,0 +1,1 @@
+> consoleProject

